### PR TITLE
Added language to encourage survey participation

### PR DESCRIPTION
**Pull request by kariljordan**
One of my actions was to draft language instructors can use to encourage learners to complete our surveys. I've added this language to the e-mail templates.

### CODE REVIEW

**Comment by kariljordan** [comment]
@ErinBecker @tracykteal reminder to review

**Comment by weaverbel** [comment]
Hi @tracykteal @kariljordan I can merge this PR but wonder if the material might be better posted in the Carpentries Commons repo https://github.com/carpentries/commons . The aim of that repo is to provide a place for reusable texts. Up to you - I have checked the page and it is ready to merge though I have a question of how people will find this page as it doesn't seem as if it has been linked to from a menu?

**Comment by tracykteal** [comment]
This page is where we have many email templates right now, related to the checklists 

http://www.datacarpentry.org/checklists/

There are a few types of 'commons' language and slides we'd like to provide. I'd like these email templates here for now. But we should probably better scope out what we want to have in 'carpentries/commons'. Right now it's mainly language people would use to describe the Carpentries or their role in Carpentries, rather than language people would use in coordinating a workshop.

File: pages/email-templates.md [hunk 1]
```diff
@@ -16,11 +16,11 @@ ____________________________________________________________________________
 ### Recruiting co-instructors
```
Comment (tracykteal) [inline pages/email-templates.md]: ### Recruiting local co-instructors for a self-organized workshop

File: pages/email-templates.md [hunk 2]
```diff
@@ -97,11 +97,42 @@ Thank you for attending our Data Carpentry workshop. We hope that you had fun an
...
 Best, 
 
 ____________________________________________________________________________
 
+
+### Encourage learners to complete pre-workshop survey
+
+Hello,
+
+We are excited to meet you at the upcoming [Data/Software/Library Carpentry workshop](INSERT WORKSHOP WEBSITE LINK). To help your instructional team provide an inclusive environment, and to learn more about your current experience with the tools you'll learn, we ask that you complete our [pre-workshop survey](https://www.surveymonkey.com/r/dcpreworkshopassessment?workshop_id=[workshop_id_value]).
```
Comment (tracykteal) [inline pages/email-templates.md]: instructional team -> instructors

just a little more standard language

**Comment by tracykteal** [comment]
One minor suggestion. Otherwise LGTM.

**Comment by kariljordan** [comment]
@tracykteal I changed "instructional team" to "instructors". Thanks Tracy!

**Pull request merged**